api/specificAccessOrchastrator/index.ts

```diff
@@ -1,8 +1,7 @@
-import logger from '@pact-foundation/pact-node/src/logger';
```

Comment:
Trying to import some random logger from the PACT dev dependency was causing issues when trying to run the app from a newly generated `yarn.lock`.